src/main/kotlin/com/jjarappappa/imom/domain/user/service/UserService.kt

```diff
@@ -20,4 +26,26 @@ class UserService (
         userFacade.validateCreateUser(request);
         userRepository.save(request.toEntity(passwordEncoder));
     }
+
+    @Transactional(readOnly = true)
+    fun login(request: UserLoginRequest): TokenResponse {
+
+        val user = userFacade.findUserByEmail(request.email)
+        checkPassword(user.password, request.password)
```

Comment:
근데 이거 checkPassword method가 앞에 게 actual이고 뒤에 게 expected라서 
```suggestion
        checkPassword(request.password, user.password)
```
이렇게 하는 게 맥락상 맞아염